doc/py_tutorials/py_imgproc/py_contours/py_contours_begin/py_contours_begin.markdown

```diff
@@ -17,23 +17,22 @@ detection and recognition.
 
 -   For better accuracy, use binary images. So before finding contours, apply threshold or canny
     edge detection.
--   findContours function modifies the source image. So if you want source image even after
-    finding contours, already store it to some other variables.
+-   Since OpenCV 3.2, findContours() no longer modifies the source image but returns a modified image as the first of three return parameters.
 -   In OpenCV, finding contours is like finding white object from black background. So remember,
     object to be found should be white and background should be black.
 
 Let's see how to find contours of a binary image:
-@code{.py}
+```python
```

Comment:
Not supported:
- prev: http://docs.opencv.org/master/d4/d73/tutorial_py_contours_begin.html
- new: http://pullrequest.opencv.org/buildbot/export/pr/8150/docs/d4/d73/tutorial_py_contours_begin.html